test/model.jl

```diff
@@ -364,8 +364,18 @@ function test_model()
         model = Model(with_optimizer(MOIU.MockOptimizer, mock))
         @test JuMP.set_silent(model)
         @test MOI.get(backend(model), MOI.Silent())
+        @test MOI.get(model, MOI.Silent())
         @test !JuMP.unset_silent(model)
         @test !MOI.get(backend(model), MOI.Silent())
+        @test !MOI.get(model, MOI.Silent())
+    end
+
+    @testset "set_parameter" begin
+        mock = MOIU.UniversalFallback(JuMP._MOIModel{Float64}())
```

Comment:
`JuMP._MOIModel` should be replaced by `MOIU.Model`

Reply:
FIxed it in 11f047af2f8422640fbae67c12bdedb21a743e48